Adding PM job description, minor tweaks to UX designer (#63)
sourcegraph/careers: job-descriptions/product-manager.md

![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)

# Product Manager

> ### About us

Reviewer: ```suggestion
## About us
```

Author: This is how headings have been done in all the other job listings. If we change it here we should also change on others for consistency. cc @mercadon and @nicksnyder for reasoning on the heading depth.

Reviewer 2: I assume this was done for visual reasons. A `##` heading is rendered with a horizontal line whereas a `###` heading is not. If we want to change this, we should do it across the board in a separate PR.

Author: There are several suggestions that would change standard language and formatting in this PR. Let's use this PR to discuss these changes and I will own making a follow up PR to make everything consistent. 

I checked rendering with the `##` heading and it looks good to me, so I will change this here and follow up with a change for all the other job descriptions.

[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.

We value openness and transparency. [Our handbook](https://about.sourcegraph.com/handbook), [our strategy](https://about.sourcegraph.com/company/strategy), and [our code](https://github.com/sourcegraph/sourcegraph) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.

To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.

If you are passionate about making the world better through software, come join us!

> ### About the role

Reviewer: ```suggestion
## About the role
```

> You will help drive the direction of Sourcegraph by collaborating with a small team to solve challenging problems that are fundamental to the growth and success of our business.

Reviewer 3: Does "small team" mean the product team or the whole Sourcegraph team?

Author: Since this person would have ownership over a piece of the product, I was envisioning both the small product team and the team of devs that they interact closely with.

Reviewer 3: Got it. I didn't understand that as it was written.

Author: What if I replaced small with cross-functional? Does that better communicate the intent to you? Or do you have another suggestion?

Reviewer 3: Here is a suggestion:

> You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.

> You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.

Reviewer: > You'll work with developers and other team members (from sales, marketing, and design) to plan, ship

Will they be shipping features?

Author: They will be integral in the process to ship changes, but are not expected to be doing the implementation themselves.

> You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.

Reviewer: ```suggestion
You'll work with engineering, sales, marketing, and design to plan, ship, and grow usage and engagement specific areas of our product. You'll have significant product ownership and gather feedback from customers to drive continuous improvement and increase engagement.
```

Author: I refactored yours a bit to the below, what do you think? I think communicating that this is a newer role for the company (early member of the team) is important for expectations as well as opportunity.

```suggestion
You'll work with engineering, sales, marketing, and design to plan, ship, and grow usage and engagement in specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our users to drive continuous improvement and increase engagement.
```

Reviewer 2: I like @christinaforney 's suggestion, with one comment: the word "engagement" stutters.

```suggestion
You'll work with engineering, sales, marketing, and design to plan, ship, and grow usage and engagement in specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our users to drive continuous improvement.
```

Reviewer: @nicksnyder's version looks good to me!

> Why this role is special:

Reviewer: Do you mean why Sourcegraph is a killer product to work on?

Author: It's why Sourcegraph is an awesome place to be a PM, so it's specific to the role. I could make it a sub-heading to the section it's in to call it out more!

Reviewer: Ahh ok. What about a heading like "Why become a product manager at Sourcegraph?"

Author: With the refactors to the below sentences, I actually think we don't need this announcement anymore. 
```suggestion
```

> - Our product has extremely high engagement relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.

Reviewer 3: Link to Yelp's blog post as evidence of the high engagement?

Author: Great idea! Added.

> - Our product has [extremely high engagement](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.

Reviewer: Because of the content length of the bullets, I'd suggest using paragraphs instead.

```suggestion
Developers [love our product](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) and rely on it daily to ship and deploy better software. Developers provide a constant stream of thoughtful feedback and love engaging with us to help them and their teams solve their toughest problems.
```

> - Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.

Reviewer: I'm not sure this bullet is needed as the previous content covers this message. 

> You won't be fighting apathy.

I think the previous sentence communicates that developers care about our product so this sentence could be removed.

> Review our [roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) to understand how our engineering team is organized and our [OKRs](https://about.sourcegraph.com/company/okrs) to understand the goals you will be helping drive.

Reviewer: ```suggestion
Our [public roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) shows the product priorities for each engineering team and we use [OKRs](https://about.sourcegraph.com/company/okrs) to communicate company goals for each quarter.
```

> #### Responsibilities

Reviewer: ```suggestion
## Responsibilities
```

You will:

> - Own an area of the product, and create and drive its product roadmap.

Reviewer: ```suggestion
- Be responsible for improving the utility and engagement for specific areas of the product.
```

Author: I'm going to leave this one as is with a small tweak create -> inform. I want to clearly communicate that they will have ownership of the product and roadmap.

> - Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.

Reviewer 2: I have a negative reaction to that word "ideate"; it seems very corporate to me. Can we avoid it or use a different word?

Author: Is brainstorm better?

Reviewer: Design?

Reviewer: Conceptualize?

Reviewer 2: Brainstorm is fine. What if we just left it as "plan and build"? or "create, plan, and build"

```suggestion
- Collaborate with engineers and designers to plan and build the solutions to customer problems.
```

> - Use customer feedback to validate priorities, improve documentation, and define product requirements.

Reviewer: ```suggestion
- Gather customer feedback to validate priorities, improve documentation, and define product requirements.
```

> - Be transparent in your internal and external communication. Sourcegraph is open source, and most discussion occurs in public, or in channels where you communicate directly with our customers.

Reviewer: ```suggestion
- Be transparent in your internal and external communication. Sourcegraph is open source with most discussions being public or in channels where you communicate directly with our customers.
```

Author: Slight tweak here. Thoughts?

```suggestion
- Be transparent in your internal and external communication. Sourcegraph is open source, so most discussions are public or in channels where you communicate directly with our customers.
```

- Be a passionate advocate for a delightful user experience.

> #### Qualifications

Reviewer: This isn't a list of qualifications per se.

```suggestion
## Required skills and experience
```

Author: We use Qualifications as a heading in all our other jobs, and it is common in most job descriptions online. I think we should keep it. I do like your intro paragraph though, so I'm going to take that :)

- Experience working as a software engineer.
- Strong understanding of Git and Git workflows.
- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
- Ability to stay focused on goals and strategically prioritize work.
- Deep passion for developer tools and productivity enhancements.
- Demonstrated leadership, the ability to drive product strategy and work well with senior engineers and designers.

> #### Nice to haves

Reviewer 3: ```suggestion
#### Nice-to-haves
```

seems clearer to me

Author: I'm fine with this, but for consistency we should update this heading in all the other job postings to have the dashes too. Should I include that in this PR?

Reviewer 3: No, it's not important.

> - Demonstrated leadership, the ability to drive product strategy and work well with senior engineers and designers.

Reviewer: ```suggestion
- Demonstrated leadership skills and ability to drive and influence product strategy across engineering, design, and distribution.
```

> - You have experience working as a product manager on a software development product.

Reviewer: ```suggestion
- Previous product management experience on a software development or developer tools product.
```

> - You have experience building developer tools.

Reviewer: ```suggestion
- Experience building and/or contributing to developer tools.
```

> - You have experience working with remote teams.

Reviewer: ```suggestion
- Experience working remotely with globally distributed teams.
```

> ### Location

Reviewer: ```suggestion
## Location
```

> We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world.

Reviewer: ```suggestion
We are a remote-first globally distributed team focussed on results and employee happiness so wherever you choose to work is up to you.
```

> We are a remote-first, globally distributed team focused on results and employee happiness, so wherever you choose to work is up to you.

Reviewer 2: This diverges a bit from what we have in other job descriptions and it would be nice to standardize.

I like your wording but wonder if it would avoid the problem mentioned in https://github.com/sourcegraph/careers/pull/34

Author: I just updated to explicitly state location and timezone don't matter. Do you like the updated wording enough for me to update on all other job descriptions?

### Compensation and benefits

> We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.

Reviewer: ```suggestion
We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like an owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
```

I really like this!

> ### Interview process

Reviewer: ```suggestion
## Interview process
```

> 1. You [apply here](<TODO(noemi)>).

Reviewer 2: ```suggestion
1. You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAC5Mi05PiZdA_k?trackingTag=careersRepository).
```

You should verify that the application form looks like how you want it to.

1. We set up a 30 minute call to chat with you about Sourcegraph to find out what you are looking for in your next role.

> 1. We give you a small case study project to see how you would approach the job at Sourcegraph.

Reviewer: ```suggestion
1. We give you a small case study project to see how you would approach product management at Sourcegraph.
```

1. We schedule 4 hours of remote interviews over video chat across multiple days.
1. We check your references.
1. We make you a job offer.

> We also expect you to be interviewing us too, so ask us any questions you have along the way!

Reviewer: ```suggestion
We expect you to be asking questions to ensure this is an environment that suits your working style and empowers you to do your best work
```

Author: I like this, updating a bit more. @nicksnyder to thumbs up on changing this for all job descriptions.

```suggestion
We expect you to be interviewing us too, so we look forward to any questions you have! We want to ensure this is an environment that suits your working style and empowers you to do your best work.
```

Reviewer 2: I would invert it to put the motivation first.

```suggestion
We want to ensure Sourcegraph is an environment that suits your working style and empowers you to do your best work, so we are eager to answer any questions that you have about us at any point in the interview process.
```

> If you aren't ready to start interviewing but are interested chat with us about anything, reach out to us [@srcgraph](https://twitter.com/srcgraph) or hiring@sourcegraph.com.

Reviewer: ```suggestion
If you're interested but aren't yet ready to interview or have questions you need answering before engaging in the interview process,  [DM us on Twitter](https://twitter.com/srcgraph) or email hiring@sourcegraph.com.
```

Author: I like this change, but before changing across all job descriptions want to check with @nicksnyder

Reviewer 2: The if/but/or structure reads awkward to me.

Here is an alternative:

```suggestion
If you're interested in Sourcegraph but aren't yet ready to apply we are still happy to connect and answer any questions that you might have: [DM us on Twitter](https://twitter.com/srcgraph) or email hiring@sourcegraph.com.
```

> **[Click here to apply](<TODO(noemi)>)**

Reviewer 2: ```suggestion
**[Click here to apply](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAC5Mi05PiZdA_k?trackingTag=careersRepository)**
```

- Be a passionate advocate for a delightful user experience.

### Qualifications

This is a very technical role and requires software engineering experience to effectively communicate with engineers and make educated decisions affecting product scope and performance at scale.

As a result, we expect:

- Experience working as a software engineer.
- Strong understanding of Git and Git workflows.
- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
- Ability to stay focused on goals and strategically prioritize work.
- Deep passion for developer tools and productivity enhancements.
- Demonstrated leadership skills and ability to drive and influence product strategy across engineering, design, and distribution.

### Nice-to-haves

- Previous product management experience on a software development or developer tools product.
- Experience building and/or contributing to developer tools.
- Experience working remotely with globally distributed teams.

## Location

> We are a remote-first, globally distributed team focused on results and employee happiness, so wherever you choose to work is up to you. We don't require you to be in any particular timezone or location.

Reviewer 2: New wording LGTM for all job descriptions. Let's link to the new team page so people can see a concrete list of teammate locations.

https://about.sourcegraph.com/company/team
